docs/docs/2020-Bismita-Guha.md

```diff
@@ -0,0 +1,195 @@
+- **What issues or roadblocks have you encountered this week?**<br/><br/>None till now<br/><br/>
+- **Do any of the issues or roadblocks still exist and what steps have been taken to resolve them?**<br/><br/>None till now<br/><br/>
+- **Is further assistance necessary to resolve existing issues?**<br/><br/>None till now<br/><br/>
+- **What do you plan to accomplish next week?** <br/><br/>1. Work on backend form models<br/> 2. Work on backend endpoints for creating forms and fields.<br/><br/>
+- **How does your progress compare to your project schedule?**<br/><br/> Done<br/><br/>
+- **Blog post for Week 3:** [Week 3 Blog](https://medium.com/@bguha/gsoc20-coding-phase-week-3-9d084ea55bdc)
+
+##
+### Weekly Status Report for Week 4
+- **What have you accomplished this week (list specific items accomplished)?**<br/><br/>1. Worked on PRs [#32](https://github.com/anitab-org/open-source-programs-web/pull/32), [#35](https://github.com/anitab-org/open-source-programs-web/pull/35), [#27](https://github.com/anitab-org/open-source-programs-backend/pull/27) & [#25](https://github.com/anitab-org/open-source-programs-backend/pull/25)<br/>2. Worked on reviews by mentors on Previous PRs<br/>3. Attended Team Meeting for OSP<br/>4. Attended Work Session with Abha and Siddharth<br/>5. Attended 1:1 with Abha, Siddharth & Monal<br/>6. Worked on Weekly Blog<br/><br/>
+- **What issues or roadblocks have you encountered this week?**<br/><br/> Previous reviews<br/><br/>
+- **Do any of the issues or roadblocks still exist and what steps have been taken to resolve them?**<br/><br/>No<br/><br/>
+- **Is further assistance necessary to resolve existing issues?**<br/><br/>No<br/><br/>
+- **What do you plan to accomplish next week?** <br/><br/>1. Start working on Phase 2 issues<br/>2. Work on mentor reviews and plan further work accordingly<br/><br/>
+- **How does your progress compare to your project schedule?**<br/><br/>On time.<br/><br/>
+- **Blog post for Week 4:** [Week 4 Blog](https://medium.com/anitab-org-open-source/form-validations-in-reactjs-4838ce32bd83?source=friends_link&sk=47929a600e688ad74dc12694703b1969)
+<hr/>
+
+## Second Coding Phase (June 30, 2020 - July 27, 2020)
+### Weekly Status Report for Week 5
+- **What have you accomplished this week (list specific items accomplished)?**<br/><br/>1. Worked on PRs [#28](https://github.com/anitab-org/open-source-programs-backend/pull/28), [#35](https://github.com/anitab-org/open-source-programs-web/pull/35) & [#36](https://github.com/anitab-org/open-source-programs-web/pull/36)<br/>2. Work ongoing on issue [#19](https://github.com/anitab-org/open-source-programs-web/issues/19)<br/>3. Attended 1:1 with May, Monal, Abha & Siddharth<br/>4. Attended Work Session with Mentors<br/>Attended Team Meeting for OSP<br/>5. Sped up the work on the Documentation<br/>6. Attended extra meeting with Monal and Abha<br/>7. Fill in First evaluation<br/><br/>
```

Comment:
@Rahulm2310  just a small change the spelling of Speed is wrong in the last line. Could you pls correct it

Reply:
5th point